routes/controllers/user.controller.js

```diff
@@ -3,38 +3,52 @@ const nodemailer = require("nodemailer");
 
 exports.getGesture = async (req, res, next) => {
   const email = req.params.users_id;
-  const user = await User.findOne({ email });
 
-  res.json({ gesture: user.gesture });
+  try {
+    const user = await User.findOne({ email });
+
+    return res.json({ gesture: user.gesture });
+  } catch (error) {
+    return next(error);
+  }
 };
 
 exports.getRecentPc = async (req, res, next) => {
   const email = req.params.users_id;
-  const user = await User.findOne({ email });
 
-  res.json({ recentPc: user.pc });
+  try {
+    const user = await User.findOne({ email });
+
+    return res.json({ recentPc: user.pc });
+  } catch (error) {
+    return next(error);
+  }
 };
 
 exports.updateRecentPc = async (req, res, next) => {
   const email = req.params.users_id;
   const recentPc = req.body.recentPc;
   const lastAccessDate = new Date();
 
-  const user = await User.findOne({ email });
+  try {
+    const user = await User.findOne({ email });
 
-  await User.updateOne(
-    { email },
-    {
-      name: user.name,
-      email: user.email,
-      gesture: user.gesture,
-      pc: {
-        name: recentPc.name,
-        ipAddress: recentPc.ipAddress,
-        lastAccessDate,
+    return await User.updateOne(
+      { email },
+      {
+        name: user.name,
+        email: user.email,
+        gesture: user.gesture,
+        pc: {
+          name: recentPc.name,
+          ipAddress: recentPc.ipAddress,
+          lastAccessDate,
+        },
```

Comment:
lastAccessDate처럼 name과 ipAddress도 비슷하게 해주면 좋을 것 같습니다!
```suggestion
        pc: {
          name,
          ipAddress,
          lastAccessDate,
        },
```